pkg/chain/ethlike/ethlikeutil/block_confirmations.go

```diff
@@ -1,13 +1,9 @@
-package chainutil
+package ethlikeutil
```

Comment:
I would consider placing those files in `pkg/chain/ethlike`. I think the reasoning behind `pkg/chain/ethereum/ethutil` was to group together components that should not be exported outside of `pkg/chain/ethereum`. In this case, we have a set of utility components that will be imported by Ethereum-like chain implementations.

Reply:
Good idea! Done in https://github.com/keep-network/keep-common/pull/64/commits/2300356d8978a9cc368ddead81e99c215d97d5d7